matlab_simulation/08-planning/example_search_graph.m

```diff
@@ -0,0 +1,47 @@
+% Get same problem every time (fix random number generator start point)
+rng('default')
+
+%% Generate a graph
+n = 30; % Nodes
+s = 100; % Space
+% Random nodes
+nodes = s*rand(n,2);
+e = sparse(zeros(n,n));
+D = sparse(zeros*ones(n,n));
+
+% Add closest p edges
+p = 5;
+for i = 1:n

```

Comment:
In the add closest p edges since it is symmetric, would it be better to go through j=i:n and not j=1:n? (line 15). That way node 1 will get the distance to all, but then node 2 will start getting the distance from node 2 to n and not need to compare it to node 1 again. Node 3 will start comparing from 3 to n etc.